lib/karafka/instrumentation/vendors/appsignal/client.rb

```diff
@@ -129,6 +145,11 @@ def stringify_hash(hash)
           def namespace_name
             @namespace_name ||= ::Appsignal::Transaction::BACKGROUND_JOB
           end
+
+          def version_4_or_newer?
+            @version_4_or_newer ||=
+              Gem::Version.new(Appsignal::VERSION) >= Gem::Version.new("4.0.0")
```

Comment:
@mensfeld we're trialling new Karafka version and see errors related to this change. I think we may use `::Appsignal::VERSION` here instead.

Reply:
aaaaaaaa fair point. Let me add integration for it and fix it. The release will go live asap

Reply:
@hieuk09 you mind monkey patching and testing if it works with the reference patch before I craft a fix and a release? :pray:

Reply:
ok added needed specs

Reply:
ref https://github.com/karafka/karafka/pull/2273

Reply:
Thanks for the quick fix. We only experience this error in development ENV so no issue so far.

Reply:
it's going to be released soon. Crafting the release as we speak

Reply:
@hieuk09 released.